pkg/model/model.go

```diff
@@ -56,29 +56,29 @@ type PowerMoldelInterface interface {
 	IsEnabled() bool
 	// GetModelType returns if the model is Ratio, LinearRegressor or EstimatorSidecar
 	GetModelType() types.ModelType
-	// GetContainerFeatureNamesList returns the list of container features that the model was configured to use
-	GetContainerFeatureNamesList() []string
+	// GetProcessFeatureNamesList returns the list of process features that the model was configured to use
+	GetProcessFeatureNamesList() []string
 	// GetNodeFeatureNamesList returns the list of node features that the model was configured to use
 	GetNodeFeatureNamesList() []string
-	// GetPlatformPower returns the total Platform Power in Watts associated to each process/container/pod
+	// GetPlatformPower returns the total Platform Power in Watts associated to each process/process/pod
 	// If isIdlePower is true, return the idle power, otherwise return the dynamic or absolute power depending on the model.
 	GetPlatformPower(isIdlePower bool) ([]float64, error)
-	// GetComponentsPower returns RAPL components Power in Watts associated to each each process/container/pod
+	// GetComponentsPower returns RAPL components Power in Watts associated to each each process/process/pod
 	// If isIdlePower is true, return the idle power, otherwise return the dynamic or absolute power depending on the model.
 	GetComponentsPower(isIdlePower bool) ([]source.NodeComponentsEnergy, error)
-	// GetComponentsPower returns GPU Power in Watts associated to each each process/container/pod
+	// GetComponentsPower returns GPU Power in Watts associated to each each process/process/pod
 	// If isIdlePower is true, return the idle power, otherwise return the dynamic or absolute power depending on the model.
 	GetGPUPower(isIdlePower bool) ([]float64, error)
 }
 
 // CreatePowerEstimatorModels checks validity of power model and set estimate functions
-func CreatePowerEstimatorModels(containerFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues []string) {
+func CreatePowerEstimatorModels(processFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues []string) {
 	config.InitModelConfigMap()
-	CreateContainerPowerEstimatorModel(containerFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
-	CreateProcessPowerEstimatorModel(containerFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
-	// Node power estimator uses the container features to estimate node power, expect for the Ratio power model that contains additional metrics.
-	CreateNodePlatformPoweEstimatorModel(containerFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
-	CreateNodeComponentPoweEstimatorModel(containerFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
+	CreateProcessPowerEstimatorModel(processFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
+	CreateProcessPowerEstimatorModel(processFeatureNames, systemMetaDataFeatureNames, systemMetaDataFeatureValues)
```

Comment:
`CreateProcessPowerEstimatorModel` is this function call duplicated?

Reply:
Thanks @vimalk78 for finding this.
I created the PR #1234 to fix it.